README.md

```diff
@@ -94,3 +94,26 @@ shell:~$ yarn start datasets compare \
 
 DiMntK7qKZQ - Z3tlf5sqWiK: equal
 ```
+
+### Copy the Organisation Units from a data set to one or more datasets:
+
+```console
+shell:~$ yarn start datasets copy-org-units \
+    --url='http://USER:PASSWORD@localhost:8080' \
+    --origin-dataset=DiMntK7qKZQ \
+    --destination-datasets=Z3tlf5sqWiK,jHF49Vvup66 \
+    [--replace, -r]
+
+shell:~$ yarn start datasets copy-org-units \
+    --url='http://USER:PASSWORD@localhost:8080' \
+    -o=DiMntK7qKZQ \
+    -d=Z3tlf5sqWiK,jHF49Vvup66 \
+    [--replace, -r]
```

Comment:
[minor] [subjective] I wouldn't duplicate the example showing the short options, the user can check this with `--help`.

Reply:
Yeah, i was unsure about that. Better to add a text note about the short options.